src/__tests__/components/introspection/GetFieldWrapper.test.tsx

```diff
@@ -0,0 +1,67 @@
+import { mockGqlSchemaIntrospection } from '@/__tests__/__mocks__/mockGqlSchemaIntrospection';
+import { GetFieldWrapper } from '@/app/components/introspection/shared/GetFieldWrapper';
+import { IIntrospectionSchema } from '@/types/introspectionImportedTypes';
+
+import { render } from '@testing-library/react';
+
+describe('GetFieldWrapper', () => {
+  it('should not render component if kind not present', () => {
+    const { container } = render(
+      <GetFieldWrapper
```

Comment:
Thank you for your tips.
At this particular case I don't see any improvements to be made by using special wrapper function.

Yes, I agree with you, that in cases like this
```tsx
    <StoreProvider>
      <LocaleProvider>
        <Suspense>
          <SomeComponentToTest />
        </Suspense>
      </LocaleProvider>
    </StoreProvider>
```

this would be useful to separate logic, and could help shorten code by not repeating it and.